src/PrestaShopBundle/Controller/Admin/Sell/Order/CartController.php

```diff
@@ -536,6 +536,10 @@ private function getErrorMessages(Exception $e)
                     'Positive product quantity is required',
                     'Admin.Notifications.Error'
                 ),
+                CartConstraintException::UNCHANGED_QUANTITY => $this->trans(
+                    'Product quantity is already the same as in cart',
```

Comment:
What action leads to this notification?

Reply:
Example:

- Product A exists in cart.

- Product A has quantity of 10.

- I search for Product A and try to add it to cart with quantity of 10.

Reply:
```suggestion
                    'Same product quantity is already in cart',
```

Reply:
Fixed in fix wording